### PR TITLE
[Feature Fix] Po collection icon fix

### PR DESCRIPTION
**Pull request by leb2dg**
# Purpose

This fixes issues created by PR #2953 . The collection icons on the project organizer are incorrect -- they are displayed as project icons. 

# Changes

Some logic was added to `resolveIconView` in *fangorn.js* so that collection icons are displayed on the project organizer.

# Side effects

In the project organizer, the collection icons appear correctly.

# Screen shots
Fix:
screen shot 2015-06-11 at 3 02 24 pm

Problem:
screen shot 2015-06-11 at 3 10 25 pm

### CODE REVIEW

File: website/static/js/fangorn.js [hunk 1]
```diff
@@ -223,6 +223,9 @@ function resolveIconView(item) {
         return returnView('link');
     }
     if (item.data.nodeType === 'project') {
+        if (item.data.parentIsFolder && item.data.isFolder && item.data.isPointer) {
+            return returnView('collection', item.data.category);
```
Comment (chennan47) [inline website/static/js/fangorn.js]: what about 
returnView('collection')
?
the second parameter is not necessary in this case.

**Comment by leb2dg** [comment]
@chennan47 Good catch! I removed the extra variable.

File: website/static/js/fangorn.js [hunk 2]
```diff
@@ -223,6 +223,9 @@ function resolveIconView(item) {
         return returnView('link');
     }
     if (item.data.nodeType === 'project') {
+        if (item.data.parentIsFolder && item.data.isFolder && item.data.isPointer) {
```
Comment (chennan47) [inline website/static/js/fangorn.js]: Why are you checking isPointer? A collection shouldn't be a pointer.

**Comment by chennan47** [comment]
Pass done. Just some minor change and this is good to go.

File: website/static/js/fangorn.js (hunk 2 above)
Comment (leb2dg) [inline website/static/js/fangorn.js]: I looked at what was in the data for the collections that were created and checked for those settings when returning the icon. I don't know why they are set like that.

**Comment by chennan47** [comment]
@sloria this is rdy to go.

**Pull request merged**